src/main/java/net/atos/zac/app/admin/HealthCheckRESTService.java

```diff
@@ -92,7 +89,7 @@ private List<ZaakType> listZaaktypes() {
 
     private RESTZaaktypeInrichtingscheck convertToREST(final ZaaktypeInrichtingscheck check) {
         final RESTZaaktypeInrichtingscheck restCheck = new RESTZaaktypeInrichtingscheck();
-        restCheck.zaaktype = zaaktypeConverter.convert(check.getZaaktype());
+        restCheck.zaaktype = RESTZaaktypeOverzichtConverter.convert(check.getZaaktype());
```

Comment:
Would these `convert` methods not be better written as extension functions?

Reply:
Well yes, but this is all Java code still. My idea was that this was a quick win in the Java code to make our life easier. Also to make it easier to convert this code to Kotlin later on.